app/recurring_job.go

```diff
@@ -574,10 +574,6 @@ func (job *Job) doRecurringBackup() (err error) {
 		return err
 	}
 
-	if err := job.doSnapshotCleanup(false); err != nil {
-		return err
-	}
```

Comment:
Do you think we can remove this? After the latest snapshot creation, the volume would contain 2 snapshots if it's not the first time creating snapshots.